src/mbgl/renderer/buckets/fill_extrusion_bucket.cpp

```diff
@@ -120,8 +120,9 @@ void FillExtrusionBucket::addFeature(const GeometryTileFeature& feature,
                     const auto d2 = convertPoint<double>(p2);
 
                     const Point<double> perp = util::unit(util::perp(d1 - d2));
-                    const auto dist = util::dist<int16_t>(d1, d2);
-                    if (edgeDistance + dist > std::numeric_limits<int16_t>::max()) {
+                    const size_t dist = util::dist<int16_t>(d1, d2);
+                    if (edgeDistance + dist >
+                        static_cast<size_t>(std::numeric_limits<int16_t>::max())) {
```

Comment:
Is this related to doxygen?

Reply:
The `core-build` failed. It is not related but I need to fix this to make the checks pass.